src/VirtoCommerce.Skyflow.Data/Providers/SkyflowPaymentMethod.cs

```diff
@@ -89,8 +90,11 @@ private async Task<PostProcessPaymentRequestResult> PostProcessPaymentAsync(Post
             {
                 throw new UnauthorizedAccessException($"Payment cannot be processed using a card registered to another user");
             }
+
+            var token = await _skyFlowClient.GetBearerToken(_options.PaymentFormAccount);
             request.Parameters["CreditCard"] = JsonConvert.SerializeObject(skyFlowCard);
             request.Parameters["ProxyHttpClientName"] = "Skyflow";
+            request.Parameters["BearerToken"] = token.AccessToken;
```

Comment:
Why do we need pass BearerToken? It should automatically added by SkyFlowhttpClient?